back-end/src/middleware/validaId.js

```diff
@@ -1,5 +1,6 @@
 const validaId = (req, res, next) => {
   const { id } = req.params;
+
   if (!Number.isInteger(+id)) {
```

Comment:
Qual o sentido dessa validação aqui?

Pergunto, pois se o valor informado no id for null, [] array vazio ou um boolean, o isInteger vai te retornar true.

Reply:
Eu não sabia desse comportamento não. obrigado!!